modules/machine-user-infra-machines-ibm-z.adoc

```diff
@@ -73,7 +73,7 @@ $ curl -LO $(oc -n openshift-machine-config-operator get configmap/coreos-bootim
 ... The gateway.
 ... The netmask.
 ... The machine host and domain name in the form `hostname.domainname`. Omit this value to let {op-system} decide.
```

Comment:
Do we want to change this _Omit this value to let {op-system} decide._ also? Seems like [anthropomorphism](https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism) also.

Reply:
@SNiemann15 You will need a follow up PR if you want to address this